llvm/tools/sycl-post-link/DeviceGlobals.h

```diff
@@ -35,7 +35,7 @@ struct DeviceGlobalProperty {
 
   // Either 1 (true) or 0 (false), telling whether the device global variable
   // was declared with the device_image_scope property.
-  uint8_t DeviceImageScope;
+  uint32_t DeviceImageScope;
```

Comment:
It would be nice to have some description why 4-bytes type is used for boolean value. Otherwise, someone may try to reduce it again in future.

Reply:
Good idea, thanks.